### ISSUE TITLE
How to print  type of variable?

### ISSUE THREAD

**Issue by hello-code** [derekparker/delve#596]
```
func main() {
    for i := 0; i < 10; i++ {
        fmt.Println("is :", i)
    }
}
```
1. What version of Delve are you using (`dlv version`)?

```
~  ᐅ dlv version
Delve Debugger
Version: 0.11.0-alpha
```
1. What version of Go are you using? (`go version`)?

```
~  ᐅ go version
go version go1.6.2 linux/amd64
```
1. What operating system and processor architecture are you using?

```
~  ᐅ uname -a
Linux d8 3.16.0-4-amd64 #1 SMP Debian 3.16.36-1 (2016-07-04) x86_64 GNU/Linux
```
1. What did you do?

```
(dlv) p i.type
Command failed: 1:3: expected selector or type assertion, found 'type'
or
(dlv) whatis i
```
1. What did you expect to see?
   `type=int`
2. What did you see instead?
   `Command failed: 1:3: expected selector or type assertion, found 'type'`

**Comment by afiune** [comment]
👍

**Comment by aarzilli** [comment]
Closed by c2092d1669b2d27054c6074c1e0d84a817b46170
